webhook.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import boto3
+import hashlib
+import hmac
+import json
+import os
+
+from base64 import b64decode
+
+
+def kms_decrypt(client, enc_value):
+    value = client.decrypt(
+        CiphertextBlob=b64decode(enc_value)
+    )['Plaintext']
+
+    return value
+
+
+kms_client = boto3.client('kms')
+
+EVENT_TYPES = ('ping', 'push',)
+SES_REGION = os.environ['SES_REGION']
+
+enc_secret = os.environ['REQUEST_SECRET']
+SECRET = kms_decrypt(kms_client, enc_secret)
+
+enc_email = os.environ['SOURCE_EMAIL']
```

Comment:
target the source e-mail is really github

Reply:
See last comment for email discussion.